mmdet/ops/roi_align/src/roi_align_cuda.cpp

```diff
@@ -19,17 +20,30 @@ int ROIAlignBackwardLaucher(const at::Tensor top_grad, const at::Tensor rois,
                             const int pooled_height, const int pooled_width,
                             at::Tensor bottom_grad);
 
+at::Tensor ROIAlign_forward_cuda_v2(const at::Tensor& input,
```

Comment:
It may be better to rearrange code similar to the original ROIAlign that contains following two functions:
`ROIAlign_forwardV2` and `ROIAlignForwardV2Laucher`.